beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/utils/MySqlTestUtils.java

```diff
@@ -50,11 +51,13 @@ public int getTableRowCount(String database, String table, String additionalFilt
     return getTableRowCount(format(SELECT_TABLE, database, table, additionalFilters));
   }
 
-  private int getTableRowCount(String statement) throws SQLException {
-    ResultSet resultSet = getTableRows(statement);
+  private int getTableRowCount(String statementString) throws SQLException {
+    Statement statement;
```

Comment:
You could combine these into the same statement. I.e. declare and initialise in the same line.